transforms/underscore-to-lodash-native.js

```diff
@@ -0,0 +1,184 @@
+const DEFAULT_OPTIONS = {
+  'split-imports': false,
+};
+
+const NATIVE_METHODS = {
+  forEach: 'forEach',
+  each: 'forEach',
+  map: 'map',
+  collect: 'map',
+  filter: 'filter',
+  select: 'filter',
+  every: 'every',
+  some: 'some',
+  find: 'find',
+  detect: 'find',
+  contains: 'includes',
+  reduce: 'reduce',
+  inject: 'reduce',
+  indexOf: 'indexOf',
+  lastIndexOf: 'lastIndexOf',
+  first: (j, identifier) => j.memberExpression(identifier, j.literal(0)),
+  last: (j, identifier) => j.memberExpression(identifier,
+    j.binaryExpression('-',
+      j.memberExpression(identifier, j.identifier('length')),
+      j.literal(1)
+    )
+  ),
+};
+
```

Comment:
> This is definitely not perfect, and it'd be super cool if we added an option if you use flowtype to use the typing data for more effective results.

Or move the less-safe mods to an option. Other "arrays" methods like `_.indexOf`, `_.lastIndexOf`, `_,findIndex` are a safer transition to built-ins.